api/cache.js

```diff
@@ -1,32 +1,32 @@
-var config = require('./config');
-var logger = require('./logging');
+const config = require('./config');
+const logger = require('./logging');
 
-var _Promise = require('bluebird');
-var redis = require('redis');
+const _Promise = require('bluebird');
```

Comment:
I think that at some point in the future it would be good to start using async/await